concepts/05 Widgets/Map/00 Overview.md

```diff
@@ -44,6 +44,62 @@ The code below adds the **Map** widget to your page. The **Map** is [centered](/
         // ...
     })
 
+##### Vue
+
+    <template>
+        <DxMap
+            :zoom="10"
+            :controls="true"
+            :center="centerCoordinates"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxMap } from 'devextreme-vue/map';
+
+    export default {
+        components: {
+            DxMap
+        },
+        data() {
+            return {
+                centerCoordinates: { lat: 40.749825, lng: -73.987963 }
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Map } from 'devextreme-react/map';
+
+    const centerCoordinates = { lat: 40.749825, lng: -73.987963 };
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+        }
+
```

Comment:
```suggestion

```

Reply:
Please make the same change to all examples in which the constructor is unused.

Reply:
fixed